test/trigger/test_trigger.js

```diff
@@ -274,6 +274,8 @@ let testValidIds = function(file, contents) {
   // as the prefix "O4" is not a full word (and have a space after it,
   // as "Prefix " does.  This is a bit rigid, but prevents many typos.
   if (ids.size > 1 && !brokenPrefixes && prefix && prefix.length > 0) {
+    // if prefix includes more than one word, just remove latter letters.
+    prefix.substr(0, prefix.indexOf(' '));
```

Comment:
I think `lastIndexOf` instead of `indexOf` will solve the multiple word problem.

Also, I think you should only do this `substr` if there is a space in the word, because otherwise `prefix` will become `undefined`.

Reply:
Ah, that's right